### PR TITLE
Make GPU plots robust to not having GPUs

### PR DESCRIPTION
**Pull request by mrocklin**
cc @quasiben @jacobtomlinson for review

### CODE REVIEW

File: distributed/dashboard/components/nvml.py [hunk 1]
```diff
@@ -134,7 +134,10 @@ def update(self):
             i = 0
 
             for ws in workers:
-                info = ws.extra["gpu"]
+                try:
+                    info = ws.extra["gpu"]
+                except KeyError:
+                    continue
                 metrics = ws.metrics["gpu"]
```
Comment (quasiben) [inline distributed/dashboard/components/nvml.py:141]: Should `ws.metrics["gpu"]` also be in the try ?

**Comment by quasiben** [comment]
This looks good to me. Thanks @mrocklin! 

What will be displayed when GPUs are not installed ?

**Comment by mrocklin** [comment]
An empty plot

**Comment by quasiben** [comment]
What do you think about including a message saying no GPUs were detected ?

**Comment by mrocklin** [comment]
I'm open to that.  We might do the same on all of the plots that show usage
like CPU usage, memory usage, GPU utilization / memory, and so on.

I'm not sure how best to do this with Bokeh though

**Comment by quasiben** [comment]
Maybe this could be done with a template like


```
{% extends main.html %}
{% block content %}
   {% if not {{GPU-FLAG}} %}
     <--GPU IS MISSING MESSAGE--> 
   {% endif %} 
{% end %}
```

Maybe this can be followed up with a subsequent PR

**Comment by mrocklin** [comment]
What if GPU machines showed up later?  I think that we would have to do
this dynamically.

**Pull request merged**